src/SignalR/samples/SignalRSamples/Startup.cs

```diff
@@ -32,17 +31,6 @@ public void ConfigureServices(IServiceCollection services)
             })
             .AddMessagePackProtocol();
             //.AddStackExchangeRedis();
-
-            services.AddCors(o =>
-            {
-                o.AddPolicy("Everything", p =>
-                {
-                    p.AllowAnyHeader()
-                     .AllowAnyMethod()
-                     .AllowAnyOrigin()
-                     .AllowCredentials();
-                });
-            });
```

Comment:
I don't know what was using this before, but now this causes the app to crash on startup, so removing it is at least better.

If we can find out how CORS is supposed to be used in the SignalRSamples app, we can redo the CORS configuration in a way that's still allowed.